controller/controller_server/csi_controller_server.py

```diff
@@ -17,6 +17,8 @@
 from controller.common.utils import set_current_thread_name
 from controller.common.node_info import NodeIdInfo
 from controller.array_action.array_mediator_action import map_volume, unmap_volume
+from controller.array_action import messages
+from controller.controller_server.config import OBJECT_TYPE_NAME_VOLUME, OBJECT_TYPE_NAME_SNAPSHOT
```

Comment:
move it upwards, right before `from controller.controller_server.errors`, or even better - use "Optimize Imports" (Ctrl+Alt+O).

you can also avoid this long extra import altogether, by using a slightly longer reference, e.g. `config.OBJECT_TYPE_NAME_VOLUME`